app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepositoryValidationResult.kt

```diff
@@ -0,0 +1,5 @@
+package com.hamdan.forzenbook.data.repository
+
+enum class LoginRepositoryValidationResult {
```

Comment:
The name of the enum shouldn't include the repository

Reply:
so then how do I differentiate it from the other enum class I will have to make on the next adjacent layer

Reply:
what should the nomenclature be like

Reply:
Try and think if you really need two layers of Errors like this. Do you? Or can the Use case just catch it?

Reply:
ok so I guess its fine to return the code as an int from the repository to the usecase, then have the use case catch it because that's where the data transformation should really be happening anyway. Ok that makes sense. The name for the enum should just be something like ValidationResult still. I just don't need it in two places.